elide-graphql/src/main/java/com/yahoo/elide/graphql/ModelBuilder.java

```diff
@@ -242,12 +242,13 @@ private GraphQLObjectType buildQueryObject(Class<?> entityClass) {
                 .name("_node__" + entityName);
 
         String id = dictionary.getIdFieldName(entityClass);
-
-        /* our id types are DeferredId objects (not Scalars.GraphQLID) */
-        builder.field(newFieldDefinition()
-                .name(id)
-                .dataFetcher(dataFetcher)
-                .type(GraphQLScalars.GRAPHQL_DEFERRED_ID));
+        if (id != null) {
```

Comment:
Our APIs require an ID.  Not sure this is the best approach.  Let's discuss.

Reply:
This change is made before we decide that we keep using row number as the id for analyticViews.